db/schema.rb

```diff
@@ -173,7 +174,7 @@
     t.integer  "duration",                default: 0,         null: false
     t.integer  "lines",                   default: 0,         null: false
     t.text     "notes"
-    t.integer  "transcript_status_id",    default: 0,         null: false
+    t.integer  "transcript_status_id",    default: 1,         null: false
```

Comment:
hmm, i didn't see this in any migration? unintended difference between this schema and your local db structure?